prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -122,7 +122,6 @@ elements:
   - name: observation-abnormal-flag
     resource: '%resource.interpretation'
     constants:
-      versionId: '2.7'
```

Comment:
Need to work this and others like this into sender transforms for SimpleReport or receiver transforms for CA.